src/main/java/com/redhat/devtools/lsp4ij/features/completion/LSPCompletionContributor.java

```diff
@@ -136,11 +143,23 @@ private void addCompletionItems(@NotNull CompletionParameters parameters,
             // Create lookup item
             LookupElement lookupItem = completionFeature.createLookupElement(item, context);
             if (lookupItem != null) {
-               completionFeature.addLookupItem(completionPrefix, result, lookupItem, size- i, item);
+                completionFeature.addLookupItem(completionPrefix, result, lookupItem, size - i, item, caseSensitive);
             }
         }
     }
 
+    private static boolean isCaseSensitive(@NotNull LanguageServerItem languageServer, @NotNull PsiFile file) {
+        LanguageServerDefinition serverDefinition = languageServer.getServerDefinition();
+        if (serverDefinition instanceof UserDefinedLanguageServerDefinition languageServerDefinition) {
+            Map<String, ClientConfigurationLanguageSettings> clientConfiguration = languageServerDefinition.getLanguageServerClientConfiguration();
```

Comment:
This code must not be here but in the custom implementation for user defined of ClientFeatures which will register a custom CompletionFeatures